brainlit/algorithms/trace_analysis/fit_spline.py

```diff
@@ -178,15 +180,6 @@ def __fit_spline_path(self, path):
 
         for row, node in enumerate(path):
             x[row, :] = self.nodes[node]["loc"]
-        orig = x.shape[0]
```

Comment:
why delete this?

Reply:
Because duplicate nodes are already prohibited at the [beginning of the function](https://github.com/neurodata/brainlit/blob/CTalgCodCov/brainlit/algorithms/trace_analysis/fit_spline.py#L104-L109), the warning will never be triggered in the middle of the operation.

Reply:
ok, can you please add a note to the docstring here that path should not have repeated nodes?

Reply:
I added a [few words](https://github.com/neurodata/brainlit/blob/CTalgCodCov/brainlit/algorithms/trace_analysis/fit_spline.py#L26) at the end of the docstring.